### ISSUE TITLE
issue using serializaer with base64

### ISSUE THREAD

**Issue by ideaBlender** [Kotlin/kotlinx.serialization#341]
Did not write any custom seriaizlation on the dto referenced below
`KJSON.nonstrict.parse(BlogPostDTO.serializer(), JSON.stringify(it))`

error:

> RangeError: Maximum call stack size exceeded
    at String_2 (string.kt?34d1:16)
    at Parser.takeStr (JsonParser.kt?2349:150)
    at JSON$JsonInput.decodeString (JSON.kt?c706:337)
    at JSON$JsonInput.ElementValueDecoder.decodeStringElement_3zr2iy$ (ElementWise.kt?1fbb:138)
    at BlogPostDTO$$serializer.deserialize_nts5qn$ (common-js.js:669)
    at decode_0 (Core.kt?6899:84)
    at JSON_0.parse_awif5v$ (JSON.kt?c706:43)
    at eval (BlogPostService.kt?8c11:41)
    at eval (RestModelService.kt?26f4:73)

data: <p><img src="data:image/jpeg;base64,/9j/4Q4wRXhpZ....

can handle everything i have thrown at it except for LARGE base64 images, small ones work great...

I would like to help if i can, pls let me know what you need.

**Comment by ideaBlender** [comment]
I will give u all the specifics when I get back to the office. Would removing that from the constructor fix it?I have a really neat kotlin-react multiplatform app and I just got quill working with images, lolOn Jan 23, 2019 10:27 AM, Leonid Startsev <notifications@github.com> wrote:This is probably the same as #314 and https://youtrack.jetbrains.com/issue/KT-29003
Which kotlin version do you use?

**Comment by ideaBlender** [comment]
Here is the dto being serialized, content is where those huge string values are (its a blog, images are stored in this field as well as base64

```
@Serializable
data class BlogPostDTO(var title: String,
                       var description: String,
                       var author: DtoMin,
                       var comments:List<DtoMin> = mutableListOf(),

                       var status: String
) : DTO(), Validateable
```

Any ideas on a quick work around

**Comment by ideaBlender** [comment]
sandwwraith the 1.3.20 fixed it without 0.10.0, so i was wrong, it was not a serialization problem, lol.  Thank you for your help!
